terminal/src/lib.rs

```diff
@@ -385,6 +385,7 @@ pub mod components {
     pub fn secret_input_with_prompt(prompt: &str) -> SecUtf8 {
         SecUtf8::from(
             Password::with_theme(&theme())
+                .allow_empty_password(true)
```

Comment:
Let's not include this here

Reply:
Hm.. I'm using a password-less local keystore and need this, any reason why we can't include this? It is in its own separate commit, for future reference.

Reply:
Ok, I guess it allows users to have lower security if they wish, but we can eventually improve on this later.